powsybl-geo-data-model/src/main/java/infrastructure/BranchGraphic.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package infrastructure;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.*;
+
+import java.util.ArrayDeque;
+import java.util.Deque;
+
+/**
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@ToString(exclude = "line")
+public class BranchGraphic {
```

Comment:
A general comment about all *Graphic classes. Once you will have replaced IIDM extensions by incubator ones (SubstationPosition and LinePosition) I recommend to use *GeoData naming which is better than graphic. 
So BranchGraphic => BranchGeoData
SubstationGraphic => SubstationGeoData
etc